src/objects.h

```diff
@@ -10381,6 +10373,7 @@ class Float32x4: public JSObject {
   // Helpers.
   static const int kLanes = 4;
   inline float getAt(int index);
+  inline float get_lane(int lane) {return getAt(lane);}
   inline float x() { return getAt(0); }
```

Comment:
I think it would be good to use lane-index-based access in our code, as the x, y, z, w concepts have been removed from spec.

I suggest to change method name `getAt` to `getLane`. Remove `get_lane` and `x()/y()/z()/w()`. It makes the interface clear.